gnovm/tests/stdlibs/std/std.gno

```diff
@@ -0,0 +1,12 @@
+package std
+
+func AssertOriginCall()                        // injected
+func IsOriginCall() bool                       // injected
+func TestCurrentRealm() string                 // injected
+func TestSkipHeights(count int64)              // injected
+func ClearStoreCache()                         // injected
+func GetCallerAt(n int) Address                // injected
+func TestSetOrigCaller(addr Address)           // injected
+func TestSetOrigPkgAddr(addr Address)          // injected
+func TestSetOrigSend(sent, spent Coins)        // injected
+func TestIssueCoins(addr Address, coins Coins) // injected
```

Comment:
This is amazing because it will also facilitate the writing of tests for those functions. I started a [PR][https://github.com/gnolang/gno/pull/1048] to tests some of them, I'll have to adapt it but that's cool.

Reply:
Oh just realize these are the test override version. My comment below was more adapted for the production version :)